### PR TITLE
Added missing transaction in SQLite query interface

### PR DESCRIPTION
**Pull request by davidrapin**
#5626 Made SQLite query-interface use the full options-object (including transactions).

### CODE REVIEW

File: lib/dialects/sqlite/query-interface.js [hunk 1]
```diff
@@ -29,14 +29,16 @@ var removeColumn = function(tableName, attributeName, options) {
...
     delete fields[attributeName];
 
     var sql = self.QueryGenerator.removeColumnQuery(tableName, fields)
       , subQueries = sql.split(';').filter(function(q) { return q !== ''; });
 
+    var rawOptions = Utils._.clone(options);
+    rawOptions.raw = true;
     return Promise.each(subQueries, function(subQuery) {
-      return self.sequelize.query(subQuery + ';', { raw: true, logging: options.logging });
+      return self.sequelize.query(subQuery + ';', rawOptions);
```
Comment (mickhansen) [inline lib/dialects/sqlite/query-interface.js]: `_.assign({raw: true}, options)` is a bit cleaner.

Comment (davidrapin) [inline lib/dialects/sqlite/query-interface.js]: Ah I was looking for something like that, thanks! As long as it does not touch the original options object :)

Comment (mickhansen) [inline lib/dialects/sqlite/query-interface.js]: It doesn't, `assign(target, ...sources)` copies from sources to target

**Comment by davidrapin** [comment]
@mickhansen Ready to merge :)

File: lib/dialects/sqlite/query-interface.js [hunk 2]
```diff
@@ -100,8 +102,9 @@ var renameColumn = function(tableName, attrNameBefore, attrNameAfter, options) {
     var sql = self.QueryGenerator.renameColumnQuery(tableName, attrNameBefore, attrNameAfter, fields)
       , subQueries = sql.split(';').filter(function(q) { return q !== ''; });
 
+    var rawOptions = Utils._.assign({raw: true}, options);
```
Comment (mickhansen) [inline lib/dialects/sqlite/query-interface.js]: No reason for a intermediate variable.

**Comment by mickhansen** [comment]
Just a single comment, please squash to a single commit after.

**Comment by davidrapin** [comment]
Ok, fixed and squashed.

**Pull request merged**